src/index.js

```diff
@@ -24,22 +24,30 @@ import {
 import type {Element} from 'react';
 import type {Context, Token} from 'fusion-core';
 
+import {InMemoryCache} from 'apollo-cache-inmemory';
+
 import serverRender from './server';
 import clientRender from './client';
 
 type ApolloClientType = mixed;
 
-export type ApolloClient<TInitialState> = (
+type TCache = mixed
+
+export type ApolloClient<TStateOrCache> = (
   ctx: Context,
-  initialState: TInitialState
+  cache: TCache,
 ) => ApolloClientType;
 
-export const ApolloClientToken: Token<ApolloClient<mixed>> = createToken(
-  'ApolloClientToken'
+export const GetApolloClientToken: Token<ApolloClient<mixed>> = createToken(
```

Comment:
Thanks for renaming this, I said we should do so, but not sure we want to introduce this yet because it would be a breaking change. (I'm not sure it's worth releasing a breaking change just for this)

Reply:
I think the whole change is worth it also. There's a reason for it to change ?

Reply:
Naming should be conventional , so this change is good and unavoidable also. Change early is better for latter development also.